de/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="de">
+<head>
+  <title>Frag nicht zum Fragen, frag einfach</title>
+  <meta property="og:title" content="Frag nicht zum Fragen, frag einfach" />
+  <meta property="og:type" content="website" />
+  <meta property="og:url" content="https://dontasktoask.com" />
+  <meta property="og:locale" content="de_DE" />
+  <link rel="stylesheet" type="text/css" href="/style.css">
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+</head>
+<body>
+  <main>
+    <h1>Frag nicht zum Fragen, frag einfach</h1>
+    <p>
+      Seitdem ich im Internet unterwegs bin, gibt es immer irgendwelche
+      Leute, die sowas schreiben:

```

Comment:
```suggestion
      Leute, die folgendes schreiben:
```

"Sowas" ist  umgangssprachlich.